ql/src/java/org/apache/hadoop/hive/ql/exec/FunctionRegistry.java

```diff
@@ -523,7 +523,6 @@
     system.registerGenericUDF("from_utc_timestamp", GenericUDFFromUtcTimestamp.class);
     system.registerGenericUDF("to_utc_timestamp", GenericUDFToUtcTimestamp.class);
 
-    system.registerGenericUDF("unix_timestamp", GenericUDFUnixTimeStamp.class);
     system.registerGenericUDF("to_unix_timestamp", GenericUDFToUnixTimeStamp.class);
```

Comment:
instead of keeping `GenericUDFToUnixTimeStamp` I think we should rename `GenericUDFToUnixTimeStamp` to `GenericUDFUnixTimeStamp` ; because that's more logical